### ISSUE TITLE
Caching of a page or the ability to download save them locally

### ISSUE THREAD

**Issue by PigeonCoding** [Ashinch/ReadYou#629]
**1. Environment**

-   Device: [Poco m3]
-   OS: [Android 13]
-   APP: [0.9.11]

**2. Describe the feature request**

Hello I would like to know if it would be possible to add a way to save an article to read later or just to have it available later when offline

**Comment by Ashinch** [comment]
Someone has mentioned this suggestion to me before, but due to time and priority constraints, I only completed half of the work. If there’s an opportunity later, I’ll continue to refine it.

**Comment by PigeonCoding** [comment]
Yeah of course just wanted to know if it was possible and since it's planned I'll just wait hopefully you can add it
Thank you for your amazing work
